File: GO/Blobs_test.go
/**
 * @version GO
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ go test
 */
package GO

import "testing"

func TestBlobs(t *testing.T) {
	tables := []struct {
		N    float64
		want int
	}{
		{40.0, 6},
		{200.0, 8},
		{300.0, 9},
	}

	for _, table := range tables {
		got := countDays(table.N)

		if got != table.want {
			t.Errorf("Waiting for this %d but the return was this: %d", table.want, got)
		}
	}
}

File: JS/Blobs.test.js
/**
 * @version JAVASCRIPT
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ node NomeDaClasse.test
 */
 const TEST = require('tape')
 const OBJ = require('./Blobs.js')
 
 TEST('Starting Blobs test...', (t) => {
     //Test 1
     t.assert(OBJ.countDays(40.0) === 6, "Expect: 40.0 = 6")
     //Test 2
     t.assert(OBJ.countDays(200.0) === 8, "Expect: 200.0 = 8")
     //Test 3
     t.assert(OBJ.countDays(300.0) === 9, "Expect: 300.0 = 9")
 
     t.end()
 })
 
